api/app/responses/api.py

```diff
@@ -306,10 +311,18 @@ def get(self, event_id):
             r.reviewer_user_id: r for r in review_responses
         }
 
+        event = event_repository.get_by_id(event_id)
+        
         serialized_responses = []
         for response in responses:
-            reviewers = [_serialize_reviewer(r, reviewer_to_review_response.get(r.reviewer_user_id, None)) 
-                         for r in response_to_reviewers.get(response.id, [])]
+            reviewers = []
+            action_editor = None
+            for r in response_to_reviewers.get(response.id, []):
+                if not r.is_action_editor:
+                    reviewers.append(_serialize_reviewer(r, reviewer_to_review_response.get(r.reviewer_user_id, None)))
+                else:
+                    action_editor = _serialize_reviewer(r, reviewer_to_review_response.get(r.reviewer_user_id, None))
```

Comment:
Nit: Since the line of code is fairly complex, would extract another variable here to make it cleaner (also a very minor comment that's it's generally easier to read "positive" conditions rather than negative ones) 

```
reviewer = _serialize_reviewer(r, reviewer_to_review_response.get(r.reviewer_user_id, None))
if r.is_action_editor:
  action_editor = reviewer
else:
  reviewers.append(reviewer)
```